src/Rules/FunctionCallParametersCheck.php

```diff
@@ -607,10 +617,19 @@ private function processArguments(
 				&& !$parameter->isVariadic()
 				&& !array_key_exists($parameter->getName(), $unusedParametersByName)
 			) {
-				$errors[] = RuleErrorBuilder::message(sprintf('Argument for parameter $%s has already been passed.', $parameter->getName()))
-					->identifier('argument.duplicate')
-					->line($argumentLine)
-					->build();
+				if ($nonUnpackAfterUnpacked) {
+					$errors[] = RuleErrorBuilder::message(sprintf('Named parameter cannot overwrite already unpacked argument $%s.', $parameter->getName()))
```

Comment:
I'd argue we don't need to change this at all. The current error message (argument.duplicate) already describes the situation correctly.

Reply:
Interesting point. I think the only reason I'm doing this was to have a non-ignorable error because it is a fatal runtime one. But maybe it's not worth the complexity?

Reply:
I only make errors non-ignorable where I'm certain about it. Here we might depend on info from PHPDoc's array shapes etc. so it's not certain - it should be ignorable.